src/components/views/messages/RedactedBody.tsx

```diff
@@ -21,8 +21,13 @@ import MatrixClientContext from "../../../contexts/MatrixClientContext";
 import { formatFullDate } from "../../../DateUtils";
 import SettingsStore from "../../../settings/SettingsStore";
 import { IBodyProps } from "./IBodyProps";
+import { MatrixEvent } from "matrix-js-sdk/src/models/event";
```

Comment:
The external import tend to be bumped at the top of the list

Reply:
We need an eslint rule for this :P

Reply:
Ah definitely agree with that.
It appears that there's some ways to do this, https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md